distribution/lib/Standard/Table/0.0.0-dev/src/Errors.enso

```diff
@@ -144,6 +144,18 @@ type Invalid_Aggregation
     to_display_text self =
         "The "+self.column+" could not be calculated at "+self.rows.short_display_text+": "+self.message
 
+## Indicates that Nothing values were found and ignored.
```

Comment:
I assume this will also ignore `NaN`s?

Reply:
I'll add tests for NaNs

Reply:
We probably need to update the warning message to sth like `Nothing/NaN` values I assume? (At least for `Float` type, as for `Integer` NaN is of course not applicable).

Reply:
Done